test/factories/users.rb

```diff
@@ -0,0 +1,80 @@
+require "factory_bot_rails"
+require "faker"
+
+FactoryBot.define do
+  factory :user do
+    first_name { Faker::TvShows::SiliconValley.character.split.first }
+    last_name { Faker::TvShows::SiliconValley.character.split.last }
+    password { "secret" }
+
```

Comment:
we need to add these as well so sidekiq doesn't try to kick off mailer jobs and so that the user accounts are confirmed, otherwise they cannot login
```
confirmation_sent_at { Time.current }
confirmed_at { Time.current }```

Reply:
Good catch! Will update this.